### DISCUSSION TITLE
06/13 안건

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#79] [Announcements]
## 이슈 다시 오픈 & 새로운 이슈 생성
상황)  Icon 컴포넌트에 버그를 남경, 유현이 발견
- 남경은 기존 이슈를 다시 오픈해서 고치면 되므로 디스커션에 올려 알리는 방법으로 생각
- 유현은 버그라고 생각해서 이슈로 생성

✅ 기준
 - 다시 오픈 → 기존 이슈에서 구현해야 할 기능(주로 커밋 타입이 feat에 어울리는 기능들)
 - 새로운 이슈 → 로직 오류 (주로 커밋 타입이 fix에 어울리는 기능들)

<br />

## Discussions 안건들 정리
- 안건에서 정해진 내용들을 찾기가 어려운 문제가 있음
  - ✅ 하나의 안건은 하나의 idea label의 discussion 게시글로 작성하고 06/14 안건 게시글에 목록으로 나열

## 안건들

-  #82 
- ✅ SkillIcon 공통 컴포넌트에 사용되는 props.name #83 
- Hooks도 스토리북 만들어야 할지?
   - ✅ 간단한 사용 방법을 알 수 있도록 스토리북 작성
- ✅ #84
- ✅ #2 에서 정해진 C/C++ 하나의 변수명과 Figma에 기술뱃지 C, C++ 2개인 점 (#86)
- ✅ #87
- ✅ #88

**Comment by Hyevvy** [comment]
혜경은 기존 이슈에서 빠진 내용이라면 reopen을 하는게 좋을 거 같고, 버그라면 새로운 issue가 나을 거 같습니다 ! 

Discussion은 라벨링을 하는게 어떨까요 ??

**Comment by YuHyun-P** [comment]
**Discussions 안건들 정리**
- 혜경님이 하신 것처럼 안건 하나를 discussion 글로 만들어서 06/13 안건 글에는 안건 목록들과 링크만 나열하는 것도 좋을 것 같네요

**Reply by Hyevvy** [reply]
그것도 좋네용 !! 😳
